src/api/app/controllers/webui/project_controller.rb

```diff
@@ -655,7 +655,9 @@ def load_project_info
 
     reqs = @project.open_requests
     @requests = (reqs[:reviews] + reqs[:targets] + reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
-
+    @incoming_requests =  (reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
+    @outgoing_requests = (reqs[:targets] +reqs[:reviews]).sort!.uniq
```

Comment:
Outgoing requests are requests for which the source project is the project (`@project` in this context)